packages/react-paypal-js/src/v6/hooks/usePayLaterOneTimePaymentSession.ts

```diff
@@ -46,10 +47,12 @@ export function usePayLaterOneTimePaymentSession({
 
     // Separate error reporting effect to avoid infinite loops with proxyCallbacks
     useEffect(() => {
-        if (!sdkInstance) {
+        if (sdkInstance) {
```

Comment:
This change prevents an error when there's no `sdkInstance` because the instance is still being loaded. I put this change in all of the session hooks.